README.md

```diff
@@ -48,7 +50,7 @@ To report a build event to Faros specify `deploy` in the `model` parameter and t
     status: Queued                           # possible values - Canceled, Failed, Queued, Running, Success
 ```
 
-> :clipboard: Note: If you have both the report build and report deployment steps in the same workflow you can use the outputs from the report build step as the inputs for the deployment step build parameters.
+> :clipboard: Note: If you have both the report build and report deployment steps in the same workflow you can use the outputs from the report build step as the inputs for the deployment step build parameters, e.g. `${{ steps.emit-build-info.outputs.build-id }}`
```

Comment:
Should we say that all available outputs can be found in `action.yml` or is that obvious for anyone using a GHA?

Reply:
I will mention it.